lib/configure.js

```diff
@@ -17,10 +15,10 @@ module.exports = function Configure(_basedir, _configOverride) {
   log('_basedir %s, _configOverride %o', _basedir, _configOverride);
   let basedir, configOverride;
   //settle arguments
-  basedir = (arguments.length && typeof arguments[0] === 'string') ? arguments[0] : process.env.nemoBaseDir || undefined;
-  configOverride = (!basedir && arguments.length && typeof arguments[0] === 'object') ? arguments[0] : undefined;
-  configOverride = (!configOverride && arguments.length && arguments[1] && typeof arguments[1] === 'object') ? arguments[1] : configOverride;
-  configOverride = (!configOverride) ? {} : configOverride;
+  basedir = arguments.length && typeof arguments[0] === 'string' ? arguments[0] : process.env.nemoBaseDir || undefined;
```

Comment:
ESlint is auto fixing this as per the eslint:recommended rules. There 4 rules on ternary operators but non of them says about parentheses.